src/strategies/erc721-pair-weights/index.ts

```diff
@@ -0,0 +1,48 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'arpitkarnatak';
+export const version = '0.1.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const calls: any[] = [];
+  options.registries.forEach((registry) => {
```

Comment:
Limiting it to 2 registries for now (as we are using only 2 balances below code), so others won't be able to pass too many registries to break something, maybe in the future, we can increase :)
```suggestion
  options.registries.slice(0,2).forEach((registry) => {
```